contracts/PostersSeason2ByMintSongs.sol

```diff
@@ -86,12 +109,33 @@ contract PosterFactory is ERC1155OnChainMetadata, IERC2981Upgradeable {
             _imageUri,
             initialSupply,
             _maxSupply,
-            _royaltyRecipient
+            _royaltyRecipient,
+            _media_type
         );
         poster[id] = newPoster;
         return id;
     }
 
+    /**
+     * @dev airdrop 1 free poster
```

Comment:
is this considered airdropping given they are collected directly against the contract?

Reply:
whoops good catch, now that we've removed the `_to` param it's not an airdrop.